tf_patches/stream_executor.diff

```diff
@@ -28,3 +28,112 @@ index 722fa93147b..16790810d1f 100644
  )
 
```

Comment:
This patch is for https://github.com/openxla/xla/commit/6d8b5c0d98aad645debbaddcb9f332ff22c19581, will delete this patch after pin updated to date after May09

Reply:
Can you move the xla.bzl change to a new file and leave a comment on the top saying to remove it after the next update?

e.g. https://github.com/pytorch/xla/blob/master/tf_patches/profiler_trace.diff

Reply:
Thanks, Will, updated